deploy/k8s/charts/trustification/templates/services/report/030-PersistentVolumeClaim.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.modules.report.enabled }}
+{{- $mod := dict "root" . "name" "report-server" "component" "report" "module" .Values.modules.report -}}
+
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: report-server
+  labels:
+    {{- include "trustification.common.labels" $mod | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "20"
+    "helm.sh/hook-delete-policy": before-hook-creation
```

Comment:
Do we really need to delete the PVC with every deployment?

Reply:
ok, I will remove it.